sky/adaptors/aws.py

```diff
@@ -56,7 +56,7 @@ def session():
 
 
 @import_package
-def client_exception():
+def botocore_exceptions():
     """Client exception."""
```

Comment:
nit: update docstring to `"""botocore exceptions."""`? (Kind of redundant though, feel free to remove too)

Reply:
Good catch! Fixed. Thanks!